src/wallet/wallet2.h

```diff
@@ -1201,7 +1203,18 @@ namespace tools
       a & m_scanned_pool_txs[1];
       if (ver < 20)
         return;
-      a & m_subaddresses.parent();
+      if (ver < 31)
```

Comment:
Not a fan of magic numbers. Give 31 a descriptive name. Would make the code more newbie friendly.